language/analysis/src/main/java/com/google/cloud/language/samples/Analyze.java

```diff
@@ -164,19 +170,42 @@ public Analyze(LanguageServiceClient languageApi) {
   }
 
   /**
+   * Gets {@link Entity}s from the string representing the GCS {@code path}.
```

Comment:
Perhaps:

    * Gets {@link Entity}s from the contents of the object at the given GCS {@code path}

Otherwise it sounds like it's looking for entities in the path string itself. (here and below)